### PR TITLE
remove outdated https client warning in docs #645

### PR DESCRIPTION
**Pull request by note**
resolves https://github.com/akka/akka-http/issues/645

### CODE REVIEW

**Comment by jrudolph** [comment]
OK TO TEST

File: docs/src/main/paradox/java/http/client-side/client-https-support.md [hunk 1]
```diff
@@ -56,7 +51,7 @@ Akka HTTP relies on [Typesafe SSL-Config](http://typesafehub.github.io/ssl-confi
 things related to SSL/TLS much simpler than using the raw SSL APIs provided by the JDK. Please refer to its
 documentation to learn more about it.
 
-All configuration options available to this library may be set under the `akka.ssl-context` configuration for Akka HTTP applications.
+All configuration options available to this library may be set under the `akka.ssl-config` configuration for Akka HTTP applications.
```
Comment (ktoso) [inline docs/src/main/paradox/java/http/client-side/client-https-support.md]: thanks 👍

**Comment by ktoso** [comment]
Yeah, I think it's correct to remove this - it is handled properly now.

**Comment by ktoso** [comment]
Thanks @note !

**Comment by jonas** [comment]
It was a comment in the Akka Sphinx docs and I converted it into a warning in https://github.com/akka/akka-http/pull/313/commits/4255db97d2defeb271f5828a6b09b8abaf4c87f3 during the convertion to Paradox apparently without reading the text :-/

**Comment by ktoso** [comment]
Hehe, no worries @jonas, glad we caught and removed it :)

**Comment by jonas** [comment]
@note BTW, I am wondering do you get a lot of pings related to Paradox and use of `@@@note` in commit messages?

**Comment by note** [comment]
@jonas haha, actually I am getting some, but I am really accustomed that I have a lot of unwanted notifications - my username is pretty unfortunate ;) Was contacting github support long time ago in that regard but I guess it's not that easy to fix

**Pull request merged**